src/libtriton/ast/astContext.cpp

```diff
@@ -741,6 +741,15 @@ namespace triton {
         }
       }
 
+      if (this->modes->isModeEnabled(triton::modes::AST_OPTIMIZATIONS) &&
+          node->getBitvectorSize() <= 512) {
```

Comment:
`&& node->getBitvectorSize() <= 512` is always true?

Reply:
Done